### ISSUE TITLE
Low contrast boxes on new documentation webpages

### ISSUE THREAD

**Issue by bronger** [django/djangoproject.com#198]
FWIW, on my screen and seen with my eyes, the boxes on the new documentation pages have a background colour almost as bright as the main background.  This way, the contrast is reduced compared to the old pages, making quick skimming more difficult.

For instance, look at the code snippets on https://docs.djangoproject.com/en/1.7/intro/overview/ and the adminition boxes on https://docs.djangoproject.com/en/1.7/intro/install/

**Comment by timgraham** [comment]
I'm not sure if any style tweaks were made since this issue was created, but it may still be relevant. (stalebot, please don't close.)

**Comment by pauloxnet** [comment]
Maybe this issue is superseded by #974 and can be closed ?

**Comment by sabderemane** [comment]
> Maybe this issue is superseded by https://github.com/django/djangoproject.com/issues/974 and can be closed ?

I don't think so, accessibility have been improved in dark mode but light mode have still issues (I tried to keep the light mode has it was) and it's also related to the multiple shade of colors we have. @xi have started [a PR](https://github.com/django/djangoproject.com/pull/1233) which will address parts (entirely?) the issue.

**Comment by kjaymiller** [comment]
> > Maybe this issue is superseded by #974 and can be closed ?
> 
> I don't think so, accessibility have been improved in dark mode but light mode have still issues (I tried to keep the light mode has it was) and it's also related to the multiple shade of colors we have. @xi have started [a PR](https://github.com/django/djangoproject.com/pull/1233) which will address parts (entirely?) the issue.

I noticed today that when you highlight code blocks (in my case to show another developer a potential solution). The vibrant green completely washes out the muted colors and it is incredibly hard to read in dark mode . (Happy to create a separate issue on this)

![washed out code block that is highlighted by the cursor](https://github.com/user-attachments/assets/b3cfb86f-6ab0-409e-8219-0125303b7dda)

**Comment by shreyadesai20042009** [comment]
I’d like to work on improving contrast in light mode for code blocks and
admonition boxes, following WCAG accessibility guidelines.
Please let me know if that approach sounds good.
